### PR TITLE
Updating marathon file to change mount of host to container to use: '/var/lib/docker/containers'

### PR DESCRIPTION
**Pull request by rashmichandrashekar**

### CODE REVIEW

**Review by samisms — APPROVED** [review]

**Review by joerg84 — APPROVED** [review]

File: repo/packages/M/msoms/2/marathon.json.mustache [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  ],
+  "acceptedResourceRoles": [
+    "slave_public",
+    "*"
+  ],
+  "healthChecks": [
+    {
+      "protocol": "COMMAND",
+      "command": {
+        "value": "status=$(ps -ef | grep -v grep | grep omsagent | wc -l); if ($status > 0); then exit 0; else exit 1; fi"
```
Comment (kensipe) [inline repo/packages/M/msoms/2/marathon.json.mustache:60]: this is a ridiculous health check... the presence of a process doesn't make it healthy

**Review by kensipe — CHANGES_REQUESTED** [review]

File: repo/packages/M/msoms/2/marathon.json.mustache (hunk 1 above)
Comment (kensipe) [reply repo/packages/M/msoms/2/marathon.json.mustache:60]: "ridiculous" may be too strong a comment... but it doesn't make sense IMO.   Is there a better option?

Comment (rashmichandrashekar) [reply repo/packages/M/msoms/2/marathon.json.mustache:60]: This is unrelated to this pull request. Right now we dont have other options to do the health check. We'll have to address it as a separate issue.

File: repo/packages/M/msoms/2/marathon.json.mustache [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    "volumes": [
+      {
+        "containerPath": "/var/run/docker.sock",
+        "hostPath": "/var/run/docker.sock",
+        "mode": "RW"
+      },
+      {
+        "containerPath": "/var/lib/docker/containers",
+        "hostPath": "/var/lib/docker/containers",
+        "mode": "RW"
```
Comment (rashmichandrashekar) [inline repo/packages/M/msoms/2/marathon.json.mustache:18]: This is the change with respect to the mount path. This is the only update to the file.

**Review by kensipe — APPROVED** [review]
Based on changes from v1 to v2

```
✔ ~/projects/universe/repo/packages/M/msoms [pr-1834 L|✔] 
12:47 $ diff 1 2
diff 1/marathon.json.mustache 2/marathon.json.mustache
16,18c16,18
<         "containerPath": "/var/opt/microsoft/omsagent/state/containerhostname",
<         "hostPath": "/etc/hostname",
<         "mode": "RO"
---
>         "containerPath": "/var/lib/docker/containers",
>         "hostPath": "/var/lib/docker/containers",
>         "mode": "RW"
```
approving.  I strongly recommend a better HC than what is provided.

**Pull request merged**